azext_edge/edge/providers/rpsaas/adr2/command.md

```diff
@@ -0,0 +1,476 @@
+swagger:
+https://github.com/Azure/azure-rest-api-specs-pr/blob/RPSaaSMaster/specification/deviceregistry/resource-manager/Microsoft.DeviceRegistry/preview/2024-07-01-preview/deviceregistry.json
+
+## Assets
+id: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/assets/my-asset`
+
+discovered assets are a different thing but have the same props: `/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/myResourceGroup/providers/Microsoft.DeviceRegistry/discoveredAssets/my-asset`
+- the discovered asset is supposed to be created from the edge/cluster rather than cloud
+- create/update/delete are still allowed via the API
+- list via the regular vs discovered should yeild different asset lists
+
+
+Main changes:
+- datapoints are now under dataset
+- addition of topic property
+- removal of some other properties
+- somehow event/dataset requirement is no longer there?
+
+### asset
+
+  create
+
+	--asset [or --name]
```

Comment:
just --name / -n is better and fits the model we are doing for other rpaas resources.

Reply:
alrighty